pkg/os/exec.go

```diff
@@ -21,6 +22,7 @@ func RunWithPrivilege(cmdAndArgs ...string) (string, string, error) {
 }
 
 func RunWithDefaultLocale(command string, args ...string) (string, string, error) {
+	/* #nosec */
```

Comment:
Did you audit all RunWithDefaultLocale callers for that G204 issue?

Reply:
@cfergeau yes, all the callers are either preflight or from the dns side, nothing is exposed to user arg side.
```bash
$ git grep RunWithDefaultLocale
pkg/crc/oc/oc.go:       return crcos.RunWithDefaultLocale(oc.OcBinaryPath, args...)
pkg/crc/preflight/preflight_checks_darwin.go:   stdOut, stdErr, err := crcos.RunWithDefaultLocale(hyperkitPath, "version")
pkg/crc/preflight/preflight_checks_linux.go:    stdOut, stdErr, err := crcos.RunWithDefaultLocale(path, "is-enabled", "libvirtd")
pkg/crc/preflight/preflight_checks_linux.go:    stdOut, stdErr, err := crcos.RunWithDefaultLocale(path, currentUser.Username)
pkg/crc/preflight/preflight_checks_linux.go:    stdOut, stdErr, err := crcos.RunWithDefaultLocale(path, "is-active", "libvirtd")
pkg/crc/preflight/preflight_checks_linux.go:    stdOut, stdErr, err := crcos.RunWithDefaultLocale(libvirtDriverPath, "version")
pkg/crc/preflight/preflight_checks_linux.go:    stdOut, stdErr, err := crcos.RunWithDefaultLocale("virsh", "--connect", "qemu:///system", "net-list")
pkg/crc/preflight/preflight_checks_linux.go:                    stdOut, stdErr, err = crcos.RunWithDefaultLocale("virsh", "--connect", "qemu:///system", "net-dumpxml", libvirt.DefaultNetwork)
pkg/crc/preflight/preflight_checks_linux.go:    stdOut, stdErr, err := crcos.RunWithDefaultLocale("virsh", "--connect", "qemu:///system", "net-list")
pkg/crc/preflight/preflight_checks_linux.go:    stdOut, stdErr, err := crcos.RunWithDefaultLocale(path, "is-active", "NetworkManager")
pkg/crc/services/dns/dns_darwin.go:     netDeviceList, _, err := crcos.RunWithDefaultLocale("networksetup", "-listallnetworkservices")
pkg/crc/services/dns/dns_darwin.go:             stdout, stderr, _ := crcos.RunWithDefaultLocale("networksetup", "-setnetworkserviceenabled", netdevice, "off")
pkg/crc/services/dns/dns_darwin.go:             stdout, stderr, err = crcos.RunWithDefaultLocale("networksetup", "-setnetworkserviceenabled", netdevice, "on")
pkg/crc/services/dns/dns_darwin.go:                     _, _, err := crcos.RunWithDefaultLocale("ping", "-c4", ns.IPAddress)
```